docs: updates from support (#1107)
cosmos/sdk-tutorials: academy/4-my-own-chain/game-winner.md

</HighlightBox>

> To be able to terminate games you need to discern between games that are current and those that have reached an end. One of the reasons a game may have reached an end is when it was won. Therefore a good field to add is for the **winner**. It needs to contain:

Author: ```suggestion
To be able to terminate games, you need to discern between games that are current and those that have reached an end - for example, when they have been won. Therefore a good field to add is for the **winner**. It needs to contain:
```
